packages/www/components/AssetDetails/AssetDetailsBox.tsx

```diff
@@ -129,6 +141,22 @@ const AssetDetailsBox = ({ asset }: AssetDetailsBoxProps) => {
               </Cell>
             </>
           )}
+          {isClip && sourceSessionId && (
+            <>
+              <Cell css={{ color: "$hiContrast" }}>Type</Cell>
+              <Cell>Clip</Cell>
+              <Cell css={{ color: "$hiContrast" }}>Parent session</Cell>
+              <Cell>
+                <ClipButton value={sourceSessionId} text={sourceSessionId} />
+              </Cell>
+            </>
+          )}
+          {isClip && sourceAssetId && (
+            <>
+              <Cell css={{ color: "$hiContrast" }}>Parent asset</Cell>
```

Comment:
Aren't you missing the `Type` row here? Might as well move it out of the `isClip` sections, making it always visible + show the other asset types as well

Reply:
Uh right! Totally missed that, would move it out of the condition - I didn't do it at first because I didn't want to use the `isLiveStream` here as well - but yeah, that's the right thing to do, will do that in the next PR